caproto/asyncio/server.py

```diff
@@ -130,13 +137,15 @@ def __init__(self, transport):
             async def sendto(self, bytes_to_send, addr_port):
                 self.transport.sendto(bytes_to_send, addr_port)
 
-            def close(self):
-                self.transport.close()
+        udp_sock = bcast_socket()
+        try:
+            udp_sock.bind((self.host, ca.EPICS_CA1_PORT))
```

Comment:
Here we are using the user-provided host and the env-provided port, ignoring `self.port`, but this is a long-standing issue that we can address as part of #222.